app/views/gsi-iframe.html

```diff
@@ -0,0 +1,18 @@
+<body>
+  <!--
+      Publications should load this next script from:
+      https://news.google.com/swg/js/v1/swg-gaa.js
+    -->
+  <script src="<% swgGaaJsUrl %>"></script>
+
+  <script>
+    GaaGoogleSignInButton.show({
+      // Allow specific origins to get results from your Google Sign-In button.
+      allowedOrigins: ['http://localhost:8000', 'http://localhost:8080'],
```

Comment:
Should this include news.google.com i.e. SwG's production URL?

Reply:
I just added "https://scenic-2017.appspot.com", nice catch!

Reply:
Great. Last nit would be to add a comment about why these origins are needed in the SwG context but that is up to you.

Reply:
I added comments for each allowed origin